printer-creality-cr10v3-bltouch.cfg

```diff
@@ -0,0 +1,229 @@
+
+[fan]
+pin: PH6 #ar9
+
+[mcu]
+serial: /dev/serial/by-id/usb-1a86_USB_Serial-if00-port0
+#pin_map: arduino
+
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 3000
+max_z_velocity: 5
+max_z_accel: 100
+
+[display]
+lcd_type: st7920
+cs_pin: PH1 #ar16
+sclk_pin: PA1 #ar23
+sid_pin: PH0 #ar17
+encoder_pins: ^PC4, ^PC6 #^ar33, ^ar31
+click_pin: ^!PC2 #^!ar35
+
+
+[bltouch]
+sensor_pin: ^PD2 #^ar19
+control_pin: PB5 #ar11
+set_output_mode: 5V
+pin_move_time: 0.4
+#   The amount of time (in seconds) to wait for the BLTouch pin to
+#   move up or down. The default is 0.680 seconds.
+stow_on_each_sample: False
+probe_with_touch_mode: False
+#pin_up_touch_mode_reports_triggered: False 
+x_offset: 45.75
+y_offset: -3.40
+z_offset:  3.28
+samples: 2
+sample_retract_dist: 2
+samples_result: average
+
+[bed_mesh]
+speed: 50
+horizontal_move_z: 6
+mesh_min: 46.50,0.75
+mesh_max: 253.5,295.85
+probe_count: 7,7
+fade_start: 1.0
+#   The gcode z position in which to start phasing out z-adjustment
+#   when fade is enabled.  Default is 1.0.
+#fade_end: 10.0
+#   The gcode z position in which phasing out completes.  When set
+#   to a value below fade_start, fade is disabled. It should be
+#   noted that fade may add unwanted scaling along the z-axis of a
+#   print.  If a user wishes to enable fade, a value of 10.0 is
+#   recommended. Default is 0.0, which disables fade.
+#fade_target:
+#   The z position in which fade should converge. When this value is set
+#   to a non-zero value it must be within the range of z-values in the mesh.
+#   Users that wish to converge to the z homing position should set this to 0.
+#   Default is the average z value of the mesh.
+#split_delta_z: .025
+#   The amount of Z difference (in mm) along a move that will
+#   trigger a split. Default is .025.
+#move_check_distance: 5.0
+#   The distance (in mm) along a move to check for split_delta_z.
+#   This is also the minimum length that a move can be split. Default
+#   is 5.0.
+mesh_pps: 2,2
+#   A comma separated pair of integers (X,Y) defining the number of
+#   points per segment to interpolate in the mesh along each axis. A
+#   "segment" can be defined as the space between each probed
+#   point. The user may enter a single value which will be applied
+#   to both axes.  Default is 2,2.
+algorithm: bicubic
+#   The interpolation algorithm to use. May be either "lagrange"
+#   or "bicubic". This option will not affect 3x3 grids, which
+#   are forced to use lagrange sampling.  Default is lagrange.
+bicubic_tension: .2
+#   When using the bicubic algorithm the tension parameter above
+#   may be applied to change the amount of slope interpolated.
+#   Larger numbers will increase the amount of slope, which
+#   results in more curvature in the mesh. Default is .2.
+relative_reference_index: 24
+#   A point index in the mesh to reference all z values to. Enabling
+#   this parameter produces a mesh relative to the probed z position
+#   at the provided index.
```

Comment:
What is the reason for setting a RRI?

Reply:
I believe this was to get the measurement referenced in respect to the middle of printer bed, but I will remove it from the configuration too